### ISSUE TITLE
Does not work on Raspberry Pi (Arm)

### ISSUE THREAD

**Issue by ribtoks** [ankicommunity/docker-anki-sync-server#8]
#### Reproduction (what did you do - the more details you give, the better):

Installed anki sync server through docker-compose

#### What was expected to happen:

Anki sync server is working

#### What happened instead (the more details you give, the better):

Docker container kept restarting with error `standard_init_linux.go:211: exec user process caused "exec format error"` which happens because of different architectures

**Comment by kuklinistvan** [comment]
Hello,

I think it is the same as https://github.com/kuklinistvan/docker-anki-sync-server/issues/11. Can you rebuild the image on your platform and try it again?

**Comment by ribtoks** [comment]
@kuklinistvan Thanks for fast reply. Your suggestion fixes the problem - I run `build.sh` and after that it works. However, Anki 2.1.19 artifacts are not available on GitHub releases page and I'm not comfortable installing it from Mega.

Is there any way to make sync work on latest Anki for macOS?

**Comment by kuklinistvan** [comment]
You're welcome :)

Our crew is working on the latest version, see the updates here:

https://github.com/ankicommunity/docker-anki-sync-server#news

Regarding to the download, I have provided checksums for you to see if any "bonus code" has been added to the original installer. Also, I don't think the original Anki maintainers have removed the archived version, but moved it to somewhere else.

If you find it, I would appreciate if you share the new link here, and I'll also update the README.
